websites/H/Hacktoberfest/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+      "name": "Xyszx",
+      "id": "593774699654283265"
+    },
+    "service": "Hacktoberfest",
+    "description": {
+      "en": "Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. "
+    },
+    "url": "hacktoberfest.digitalocean.com",
+    "version": "1.0.3",
```

Comment:
Version still should be 1.0.0 since you are releasing presence first time.

Reply:
Okay, done.